doc/howto-devel.rst

```diff
@@ -15,14 +15,33 @@ each supported distribution::
 Vagrant
 -------
 
-**Note**: this requires installing the `vagrant-sshfs`_ plugin.
+Fork this github repostiory located at `<https://github.com/proot-me/proot>`_
+
+``git clone [YOUR_REPOSITORY_URL]`` Clone the repository on your local machine
+
+``sudo apt-get install vagrant vagrant-sshfs`` Install vagrant ( For Debian and derivatives )
```

Comment:
This is only for Debian-based distributions. Also it breaks the link to the vagrant-sshfs plugin.